aait/backend/group-2A/Application/Features/Like/Handlers/Queries/GetLikedPostRequestHandler.cs

```diff
@@ -0,0 +1,23 @@
+using Application.Contracts.Persistance;
+using Application.DTO.Post;
+using Application.Features.Like.Request.Queries;
+using AutoMapper;
+using MediatR;
+
+namespace Application.Features.Like.Handlers.Query;
+
+public class GetLikedPostRequestHandler : IRequestHandler<GetLikedPostRequest, List<PostDto>>{
```

Comment:
if you want a liked post from likeRepository wtih post id , the dto you're gonna return is not a list of postDto